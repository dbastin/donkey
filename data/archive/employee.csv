"ID","NAME","SALARY","START"
"1","Fred","1000000","2001-01-01"
"2","Wilma","2000000","2002-01-01"
"3","Barney","3000000","2003-01-01"
"4","Betty","4","null"