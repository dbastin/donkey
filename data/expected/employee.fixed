ID                      NAME
1                       Fred