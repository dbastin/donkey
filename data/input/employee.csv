ID,NAME
1,Fred
2,Wilma
3,Barney
4,Betty