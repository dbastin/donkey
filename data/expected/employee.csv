"ID","NAME"
"1000","FRED"
"2000","WILMA"
"3000","BARNEY"
"4000","BETTY"